roles/monitoring_sap/tasks/repos-activation.yml

```diff
@@ -0,0 +1,17 @@
+---
+# The role common will activate SLES OS Repos and install the prometheus packages
+
+- name: Install netcat-openbsd
+  ansible.builtin.zypper:
+    name: netcat-openbsd
+    state: present
+  tags: netcat-openbsd
+
+- name: Activate SLES repo SLE-Module-SAP-Applications15-SP5-Updates & SLE-Module-Packagehub-Subpackages15-SP5-Pool
+  ansible.builtin.command: "{{ item_cmd }}"
+  loop:
+    - hostname
+    - /usr/sbin/SUSEConnect -p sle-module-sap-applications/15.5/x86_64
```

Comment:
15.5 should not be hardcoded. Should be obtained from the system dynamically based on OS

Reply:
I wish we could. But other versions are excluded (only one package was incomplete: enough to restrict on on version 15.5)

Reply:
@AchimHaaser1 
What if version 15.6 , 15.7 comes ? Will you again hard code it ? The what about support for 15.5 ?

Reply:
changed to {{ ansible_distribution_version }}